.env.example

```
@@ -1,4 +1,4 @@
 # Configuration to be set by node owners
-EMAIL=
-CONTENT_SERVER_STORAGE=
+EMAIL=no-reply@decentraland.org
+CONTENT_SERVER_STORAGE=/srv/catalyst/storage
```

```suggestion
EMAIL=
CONTENT_SERVER_STORAGE=
```
Let's keep it as it was before